libr/asm/arch/8051/8051_disas.c

```diff
@@ -6,60 +6,7 @@
 
 #include "8051_ops.h"
 
-static const char *_8051_regs[] = {
-	"r0", "r1", "r2", "r3", "r4", "r5", "r6", "r7", // 0x00
-	0, 0, 0, 0, 0, 0, 0, 0, // 0x08
-	0, 0, 0, 0, 0, 0, 0, 0, // 0x10
-	0, 0, 0, 0, 0, 0, 0, 0, // 0x18
-	0, 0, 0, 0, 0, 0, 0, 0, // 0x20
-	0, 0, 0, 0, 0, 0, 0, 0, // 0x28
-	0, 0, 0, 0, 0, 0, 0, 0, // 0x30
-	0, 0, 0, 0, 0, 0, 0, 0, // 0x38
-	0, 0, 0, 0, 0, 0, 0, 0, // 0x40
-	0, 0, 0, 0, 0, 0, 0, 0, // 0x48
-	0, 0, 0, 0, 0, 0, 0, 0, // 0x50
-	0, 0, 0, 0, 0, 0, 0, 0, // 0x58
-	0, 0, 0, 0, 0, 0, 0, 0, // 0x60
-	0, 0, 0, 0, 0, 0, 0, 0, // 0x68
-	0, 0, 0, 0, 0, 0, 0, 0, // 0x70
-	0, 0, 0, 0, 0, 0, 0, 0, // 0x78
-	"p0", "sp", "dpl", "dph", 0, 0, 0, "pcon", // 0x80
```

Comment:
i think those register aliases shouldnt be removed. or am i missing something :?

Reply:
This part of the code is only used by replace register. And I removed replace register because it does not really points to BANK and because it break the reassembly